components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/resources/publisher-api.yaml

```
@@ -12851,12 +12863,33 @@ components:
+          type: boolean
+          description: Whether the authentication configuration is enabled or not
+          default: false
+        type:
+          type: string
+          description: Type of the authentication configuration
+          example: apiKey
+        parameters:
+          type: object
+          description: |
+            Parameters required for the authentication configuration. The parameters are different based on the type of the authentication configuration.
+          example: {"headerEnabled": true,"headerName":"Authorization"}
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)